packages/integration-tests/src/support/variables/cluster-details.ts

```diff
@@ -1,3 +1,9 @@
+import * as semver from 'semver';
```

Comment:
I don't like to set a package in a `Cypress.env`. It will be less performant and if done extensively, will load the memory with packages.

I know that's how it's done on KNI, but I don't see why the module that needs access to the package wouldn't access it directly.